src/iterative/KOSMAPOSL/KOSMAPOSLReconstruction.cxx

```diff
@@ -534,6 +541,15 @@ set_hybrid(const bool arg)
   this->hybrid = arg;
 }
 
+template <typename TargetT>
+void
+KOSMAPOSLReconstruction<TargetT>::
+set_stop_iterative_kernel_at_iter_num(const int arg)
+{
+  this->_already_set_up = false;
```

Comment:
is this necessary? i.e. do we really need to call `set_up()` again after using this?

Reply:
no probably not. I saw all the setters had this and I completely forgot why this was, so I put it just in case but can remove